src/utils/daos.c

```diff
@@ -925,6 +1012,23 @@ call_dfuse_ioctl(char *path, struct dfuse_il_reply *reply)
 	return 0;
 }
 
+static int
+fs_op_hdlr(struct cmd_args_s *ap)
+{
+	int rc = 0;
+	enum fs_op op;
+	assert(ap != NULL);
+	op = ap->fs_op;
+	switch (op) {
+	case FS_COPY:
+		fs_copy_hdlr(ap);
```

Comment:
I'm not familiar with the process here, but do we need an error code from `fs_copy`?

Reply:
I think this then brings to discussion which errors in `fs_copy` should return, and which should just be logged and continue.
I'm not sure about each one... But something to think about